model/postgres/postgres_timeline.rb

```diff
@@ -47,13 +47,13 @@ def need_backup?
 
   def backups
     blob_storage_client
-      .list_objects(bucket_name: ubid, folder_path: "basebackups_005/")
-      .select { _1.key.end_with?("backup_stop_sentinel.json") }
+      .list_objects(ubid, "basebackups_005/")
+      .select { _1.object_name.end_with?("backup_stop_sentinel.json") }
```

Comment:
Should we keep the parameters/properties of our minio client compatible  with the AWS S3 ruby client?

Reply:
solved it